packages/manager/modules/request-tagger/src/factory.js

```diff
@@ -21,14 +27,18 @@ export default /* @ngInject */ () => {
           (value, route) => new RegExp(route).test(config.url),
         ) || {};
 
-        return {
+        const headerConfig = {
           ...config,
           headers: {
             ...config.headers,
             [HEADER_REQUEST_ID]: `${Date.now()}-${requestIndex}`,
             ...overridenHeaders,
           },
         };
+        if (!isEmpty(headerVersion)) {
+          headerConfig.headers[HEADER_VERSION] = `${headerVersion}`;
```

Comment:
```suggestion
          headerConfig.headers[HEADER_VERSION] = headerVersion;
```

Reply:
Hum I think this is useful to have `2.0` written as `2.0` (as a string) instead of just `2` (as a number)